qemu/tests/driver_update.py

```diff
@@ -0,0 +1,148 @@
+import time
```

Comment:
Please adjust order of import modules.   

import os 
import time
import re
import logging